gtdynamics/kinematics/KinematicsSlice.cpp

```diff
@@ -161,4 +239,27 @@ Values Kinematics::inverse<Slice>(const Slice& slice, const Robot& robot,
 
   return optimize(graph, constraints, initial_values);
 }
+
+template <>
+gtsam::Values Kinematics::inverseWithPose<Slice>(
+    const Slice& slice, const Robot& robot, const gtsam::Values& goal_poses,
+    const gtsam::Values& joint_priors) const {
+  auto graph = this->graph(slice, robot);
+
+  // Add prior on joint angles to constrain the solution
+  graph.add(this->jointAngleObjectives(slice, robot, joint_priors));
```

Comment:
maybe "constrain" is too strong of a word for something that is just "gently encouraging a solution".  Unless I misunderstood - this is just to avoid issues w.r.t. multiple solutions in the case of a non-empty nullspace right?

Reply:
Done